operator/config/manifests/bases/rhacs-operator.clusterserviceversion.yaml

```diff
@@ -295,6 +295,11 @@ spec:
         path: misc
         x-descriptors:
         - urn:alm:descriptor:com.tectonic.ui:advanced
+      - description: Configures resources within Central in a declarative manner.
+        displayName: Declarative Configuration
+        path: central.declarativeConfiguration
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:hidden
```

Comment:
what does adding these descriptors do?

Reply:
They are called [API Markers](https://sdk.operatorframework.io/docs/building-operators/golang/references/markers/) and impact the UI that is rendered when you want to e.g. create a CR.
This marker specifically allows us to hide the field, since its currently not released and shouldn't be used by users (effectively, it won't do anything if used, but we still shouldn't show it _yet_ as prominently). We've done similar things for the telemetry values.

Additionaly,[ there are other markes like the config map / secret one](https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/components/descriptors/reference/reference.md#3-k8sresourceprefix), which automatically provide users of the UI with a selection of config maps / secrets, so its quite handy.

Reply:
Cool, today I learned something new :)